### PR TITLE
Task 1  Modify Login Web Service

### PR DESCRIPTION
**Pull request by michael-msx**
1. Modify login service to return person id.
   new result ex: {"authStatus":true,"id":1,"account":"Person","statusCode":1}
   if statusCode = -1 , the name or password is null
2. remove some IDE files from version control system.

### CODE REVIEW

File: ipcs_domain/src/main/java/com/ipcs/service/SecurityService.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
  */
 public interface SecurityService {
 
+    public long authenticateLoginInfoWithID(String userName, String password);
```
Comment (seeyou8286) [inline ipcs_domain/src/main/java/com/ipcs/service/SecurityService.java]: method authenticateLoginInfoWithID  name is misleading.
Please change name, like getAuthenticatedUserID

File: ipcs_domain/src/main/java/com/ipcs/service/impl/SecurityServiceImpl.java [hunk 2]
```diff
@@ -61,6 +61,15 @@ public void setRoleDao(RoleDao roleDao) {
         this.roleDao = roleDao;
     }
 
+
+    @Transactional
+    public long authenticateLoginInfoWithID(String userName, String password) {
```
Comment (seeyou8286) [inline ipcs_domain/src/main/java/com/ipcs/service/impl/SecurityServiceImpl.java]: method authenticateLoginInfoWithID name is misleading.
Please change name, like getAuthenticatedUserID

File: ipcs_domain/src/main/java/com/ipcs/service/impl/SecurityServiceImpl.java [hunk 3]
```diff
@@ -61,6 +61,15 @@ public void setRoleDao(RoleDao roleDao) {
...
     }
 
+
+    @Transactional
+    public long authenticateLoginInfoWithID(String userName, String password) {
+        Criterion[] criterion = { Restrictions.eq("account_name", userName) };
+        List<Person> persons = personDao.createCriteria(criterion);
+        if (null == persons || persons.size() > 1 || persons.size() == 0)
+            return -1;
+        return persons.get(0).getObjectId();
```
Comment (seeyou8286) [inline ipcs_domain/src/main/java/com/ipcs/service/impl/SecurityServiceImpl.java:71]: Please add unit test for this method.
Pls refor the unit test in test folder

**Comment by michael-msx** [comment]
change method name to getAuthenticatedUserID

**Comment by michael-msx** [comment]
Create List activities by id web service
request:  http://localhost:8080/ipcs_service/listAllActivities?id=1
result: {"id":1,"length":1,"activities":[{"name":"Math","location":"PUNGGOL","description":"math","startTime":"2015-05-20","host_account":"Teacher"}]}

**Comment by seeyou8286** [comment]
Pls dont forget to provide unitest

**Pull request merged**